src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -62,6 +62,11 @@ export default (accessToken, userID, opts) => {
       { headers: true }
     ),
     collectorProfileSummaryLoader: gravityLoader("collector_profile_summary"),
+    commerceOptInLoader: gravityLoader(
```

Comment:
#non-blocking suggestion: Its slightly better long-term if we keep the naming of things 1:1 -- if the REST endpoint is `opt_in_artworks_into_commerce`, then we should name the loader `optInArtworksIntoCommerceLoader`. Then devs using said mutation know exactly where to look in gravity should there be an issue upstream.

Reply:
ahh thanks for bringing this up! Pushed these changes in [fb9961e](https://github.com/artsy/metaphysics/pull/6039/commits/fb9961ea4c6738f27e1b8ffe0bc0a4993e6cac56).